### DISCUSSION TITLE
Usefulib Website!

### DISCUSSION THREAD

**Discussion by hamdivazim-archive** [hamdivazim-archive/usefulib#13] [Announcements]
## Introducing the new [usefulib website](https://hamdivazim.github.io/usefulib)!

Usefulib's new website has launched at https://hamdivazim.github.io/usefulib/! Please take a look and if you spot anything wrong, comment below or make an issue using the `web-issue` label!

**Comment by MKM12345** [comment]
Might I ask why we need this?

**Comment by hamdivazim-archive** [comment]
So people know what this is apart from the readme
